include/Exception.hpp

```diff
@@ -341,7 +341,7 @@ class UnknownProcedureException : public voltdb::Exception {
     }
 };
 
-class CoordinateOutOfRangeException : public voltdb::Exception {
```

Comment:
I may not have cleaned this file up and redundant namespaces not fully cleaned up for this file. I will cleanup it up. Thanks for pointing that out